src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.sarif

```diff
@@ -3645,6 +3645,63 @@
               "Telemetry"
             ]
           }
+        },
+        "CA5396": {
+          "id": "CA5396",
+          "shortDescription": "Set HttpOnly to true for HttpCookie",
+          "fullDescription": "As a defense in depth measure, ensure security sensitive HTTP cookies are marked as HttpOnly. This indicates web browsers should disallow scripts from accessing the cookies. Injected malicious scripts are a common way of stealing cookies.",
+          "defaultLevel": "warning",
+          "properties": {
+            "category": "Security",
+            "isEnabledByDefault": false,
+            "typeName": "SetHttpOnlyForHttpCookie",
+            "languages": [
+              "C#",
+              "Visual Basic"
+            ],
+            "tags": [
```

Comment:
@dotpaul @LLLXXXCCC - can you confirm the tags are correct for each of the added rules? Also do we respect https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#excluded-symbol-names for the DFA rule? If so, we should update that list of configurable rules.